interface/forms/vitals/templates/vitals/general_new.html

```diff
@@ -121,213 +134,239 @@
-                <input type="text" size='5' id='waist_circ_input_metric' value="{if $vitals->get_waist_circ() != 0}{math equation="round(number * constant,1)" number=$vitals->get_waist_circ() constant=2.54 format="%.2f"}{/if}" onChange="convCmtoIn('waist_circ_input');"/>
-                </td>
-        {foreach item=result from=$results}
-                <td class='historicalvalues'>{if $result.waist_circ != 0}{math equation="number * constant" number=$result.waist_circ constant=2.54 format="%.2f"}{/if}</td>
-        {/foreach}</tr>
-
-	<tr><td class="graph" id="BMI">{xl t="BMI"}</td><td>{xl t="kg/m^2"}</td>
-		<td class='currentvalues'><input type="text" size='5'
-			name='BMI' id='BMI_input' value="{if $vitals->get_BMI() != 0}{$vitals->get_BMI()|substr:0:5}{/if}"/></td>
-	{foreach item=result from=$results}
-		<td class='historicalvalues'>{if $result.BMI != 0}{$result.BMI|substr:0:5}{/if}</td>
-	{/foreach}</tr>
-
-  <tr><td>{xl t="BMI Status"}</td><td>{xl t="Type"}</td>
-    <td class='currentvalues'><input type="text" size='15'
-    name="BMI_status" id="BMI_status' value="{$vitals->get_BMI_status()}"/></td>
-  {foreach item=result from=$results}
-    <td  class='historicalvalues'>{if $result.BMI_status}{xl t=$result.BMI_status}{/if}</td>
-  {/foreach}
-  </tr>
-
-  <tr><td>{xl t="Other Notes"}<td></td></td>
-    <td class='currentvalues'><input type="text" size='20'
-      name="note" id='note' value="{$vitals->get_note()}" /></td>
-  {foreach item=result from=$results}
-    <td class='historicalvalues'>{$result.note}</td>
-  {/foreach}</tr>
-
-	<tr>
-	<td colspan='3' style='text-align:center'>
-	{if $patient_age <= 20 || (preg_match('/month/', $patient_age))}
-	<!-- only show growth-chart button for patients < 20 years old -->
-	<!-- <input type="button" id="growthchart" value="{xl t="Growth-Chart"}" style='margin-left: 20px;'> -->
-	<input type="button" id="pdfchart" value="{xl t="Growth-Chart"} ({xl t="PDF"})" style='margin-left: 20px;'>
-        <input type="button" id="htmlchart" value="{xl t="Growth-Chart"} ({xl t="HTML"})" style='margin-left: 20px;'>
-	{/if}
-	</td>
-	</tr>
-
-	<tr><td colspan='3' style='text-align:center'>&nbsp;</td></tr>
-	<tr>
-	<td colspan='3' style='text-align:center'>
-        <input type="submit" class="editonly" name="Submit" value="{xl t="Save"}">
-        <input type="button" class="editonly" id="cancel" value="{xl t="Cancel"}">
-        </td>
-</tr>
-</table>
-<br><br>
-<input type="hidden" name="id" id='id' value="{$vitals->get_id()}" />
-<input type="hidden" name="activity" id='activity' value="{$vitals->get_activity()}">
-<input type="hidden" name="pid" id='pid' value="{$vitals->get_pid()}">
-<input type="hidden" name="process" id='process' value="true">
-</form>
+
+ <div class="container">
+        <div class="row">
+            <div class="col-sm-12">
+                <div class="page-header">
```

Comment:
May be we should validate the behavior in various situations but if the intent is to have full row of page header div, specifying 12 columns is redundant (bs standard).

Reply:
I agree, pl remember getting the old scripts to become responsive and restyled has been a major undertaking. Improving upon the work already done would help in standardizing our layout and give developers a template/templates to use to ensure seamless integration of new scripts with the existing ones.